Flow.Launcher.Plugin/Result.cs

```diff
@@ -29,6 +31,8 @@ public class Result
         /// </summary>
         public string ActionKeywordAssigned { get; set; }
 
+        public string SuggestionText { get; set; } = string.Empty;
```

Comment:
do we want to change this to 'AutoCompleteText' to be more specific? I didnt get what it is for on first glance until i read the code. 

This is good btw, it will allow plugins to do their own autocomplete, as @onesounds have indicated, if we change how Explorer display Title and Subtitle result, this will allow Explorer to put its own autocompelete text